generate/config.toml

```
@@ -4,9 +4,10 @@ endpoints_base_file_path = "templates/base_endpoints.template.md"
 
 
 repos = [
-    "ios_repo",
-    "android_repo",
-    "service_repos",
+"ios_repo",
+"android_repo",
+"service_repos",
+"testing_repo",
```

Switch order so all mobile stuff is together
```suggestion
"testing_repo",
"service_repos",
```